packages/react-renderer-demo/src/components/code-editor/index.js

```diff
@@ -48,7 +48,18 @@ const CodeEditor = ({ value, children, className, inExample }) => {
 
   const lang = className ? className.toLowerCase().replace('language-', '') : undefined;
   let content = value || children || '';
-  content = tranformImports(content);
+
+  // read props from code in --- { "key": value } ---\n format
+  let propsFromMD = content.match(/--- .* ---/);
```

Comment:
Can we use the language to turn off the import transformation? Just disable it for diff?

Reply:
I would rather keep it - I also used it in the example in optimization page.

Reply:
OK